pkgs/tools/networking/pritunl-client/default.nix

```diff
@@ -1,27 +1,144 @@
-{ lib, buildGoModule, fetchFromGitHub }:
-
-buildGoModule rec {
-  pname = "pritunl-client";
+{ lib
+, stdenv
+, fetchFromGitHub
+, runtimeShell
+, runCommand
+, makeWrapper
+, installShellFiles
+, buildGoModule
+, coreutils
+, which
+, gnugrep
+, gnused
+, openresolv
+, systemd
+, iproute2
+, openvpn
+, electron
+, gcc-unwrapped
+}: let
   version = "1.3.3484.2";
-
   src = fetchFromGitHub {
     owner = "pritunl";
     repo = "pritunl-client-electron";
     rev = version;
     sha256 = "sha256-thIbw/Iek2vNNmkOBHrzEhTbaOm05CsbjqORQKC2jIs=";
   };
 
-  modRoot = "cli";
-  vendorHash = "sha256-miwGLWpoaavg/xcw/0pNBYCdovBnvjP5kdaaGPcRuWk=";
+  cli = buildGoModule {
+    pname = "pritunl-cli";
+    inherit version src;
+
+    modRoot = "cli";
+    vendorHash = "sha256-miwGLWpoaavg/xcw/0pNBYCdovBnvjP5kdaaGPcRuWk=";
+
+    postInstall = ''
+      mv $out/bin/cli $out/bin/pritunl-client
+    '';
+  };
+
+  service = buildGoModule rec {
+    pname = "pritunl-client-service";
+    inherit version src;
+
+    modRoot = "service";
+    vendorHash = "sha256-VuJ1giVSDlVFRJOy0UPVYp8j6TCWzXbiTgg4xyMq3dQ=";
+
+    nativeBuildInputs = [makeWrapper];
+
+    buildInputs = [
+      coreutils
+      which
+      gnused
+      gnugrep
```

Comment:
Where are those used? If they are run on the build machine then we can drop all of them except which which is going to be moved to nativeBuildInputs

Reply:
They're used by the  `openvpn` hook scripts. You can see `builtInputs` being passed to `openvpn-wrapper`